submissions/unabyband/task-js-dom/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+        <link rel="stylesheet" href="css/main.css">
+        <title>unabyband DOM</title>
+    </head>
+    <body>
+        <header>
+            <a href="index.html"><div class="home-link"></div></a>
+            <span class="inner-text">Just a little visualisation about color temperature and white balance.</span>
+        </header>
+        <div class="main-container">
+            <nav class="sidebar">
+                <div class="side-menu inner-text" id="nav-menu">  
+                    <a href="#" class="side-menu-item" id="tab-0">Clear Sky</a>
+                    <a href="#" class="side-menu-item" id="tab-1">Clouds/Shade</a>
+                    <a href="#" class="side-menu-item" id="tab-2">Midday Light</a>
+                    <a href="#" class="side-menu-item" id="tab-3">Flash/Daylight</a>
+                    <a href="#" class="side-menu-item" id="tab-4">Fluorescent</a>
+                    <a href="#" class="side-menu-item" id="tab-5">Sunrise/Sunset</a>
+                    <a href="#" class="side-menu-item" id="tab-6">Domestic Light</a>
+                    <a href="#" class="side-menu-item" id="tab-7">Candle Flame</a>
+                </div>
+            </nav> 
+            <main class="content" id="content">
+                <span class="inner-text">
+                    White balance chart from different sources of light<br><br>
+                <img src="src/wb_chart.png" alt="white balance chart"></span>
```

Comment:
`img` must have `alt` attribute.

Reply:
Definitely, and they all have "alt" attributes, even rendered from script file.